frameworks/MLNet/exec.py

```diff
@@ -0,0 +1,116 @@
+import logging
+import math
+import os
+import pandas as pd
+from amlb.benchmark import TaskConfig
+from amlb.data import Dataset
+from amlb.datautils import reorder_dataset
+from amlb.results import NoResultError, save_predictions
+from amlb.utils import dir_of, path_from_split, run_cmd, split_path, Timer
+import json
+from frameworks.shared.callee import call_run, result, save_metadata
+
+log = logging.getLogger(__name__)
+
+def run(dataset: Dataset, config: TaskConfig):
+    avaible_task_list = ['classification', 'regression']
+    if config.type not in avaible_task_list:
+        raise ValueError('{} is not supported.'.format(config.type))
+
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    DOTNET_INSTALL_DIR = os.path.join(dir_path, '.dotnet')
+    os.environ['MODELBUILDER_AUTOML'] = 'NNI'
+    os.environ['DOTNET_ROOT'] = DOTNET_INSTALL_DIR
+    mlnet = os.path.join(DOTNET_INSTALL_DIR, 'mlnet')
+    save_metadata(config)
+    name = config.name
+    temp_output_folder = os.path.join(config.output_dir,name)
+    if not os.path.exists(temp_output_folder):
+        os.mkdir(temp_output_folder)
```

Comment:
```suggestion
    # from frameworks.shared.callee import output_subdir
    temp_output_folder = output_subdir(name, config)
```
this standardizes folder structure and ensures that you get a log file for each task/fold.

Reply:
Resolved